src/librustc_driver/driver.rs

```diff
@@ -473,27 +453,29 @@ pub fn phase_2_configure_and_expand(sess: &Session,
     });
 
 
-    krate = time(time_passes, "crate injection", ||
-                 syntax::std_inject::maybe_inject_crates_ref(krate,
-                                                             sess.opts.alt_std_name.clone()));
+    krate = time(time_passes, "crate injection", || {
```

Comment:
I don't object, but any idea why this case is handled so differently from the rest, which put each arg onto its own line?